docs/src/content/hardhat-runner/docs/guides/verifying.md

```diff
@@ -46,22 +73,103 @@ module.exports = {
 
 We are going to use the [Sepolia testnet](https://ethereum.org/en/developers/docs/networks/#sepolia) to deploy and verify our contract, so you need to add this network in your Hardhat config. Here we are using [Infura](https://infura.io/) to connect to the network, but you can use an alternative JSON-RPC URL like [Alchemy](https://alchemy.com/) if you want.
 
-::::tabsgroup{options=Infura,Alchemy}
+::::tabsgroup{options="Infura | Typescript,Alchemy | Typescript,Infura | Javascript,Alchemy | Javascript"}
```

Comment:
This requires updating https://github.com/NomicFoundation/hardhat/blob/main/docs/temp/tabsConfig.json

Reply:
I was wondering why there was no tab selected by default. I'll fix it right away. Thanks for the 👀

Reply:
Done!